docs/release/trg-0/trg-4-02.md

```diff
@@ -0,0 +1,133 @@
+
+If the language or runtime environment of your product is not listed above, feel free
+to [start a discussion](https://github.com/eclipse-tractusx/sig-infra/discussions) and propose your preferred container
+images as base image.
+
+:::info
+As stated in the description above, base image usage is particularly aligned for container images, that we distribute by
+publishing them on DockerHub. In case you are using Docker images for build or testing purposes (for example _pandoc_ or
+_cypress_, etc.) and you do not publish the images, you can use other publicly available image, as long as the tools are
+open source license compliant.
+
+For automated TRG checks, you can skip base image checks on Dockerfiles by declaring it in the `.tractusx` metadata
+files. Details can be found in
+the [metadata file documentation](https://github.com/eclipse-tractusx/tractusx-quality-checks/blob/main/docs/metadata_file.md)
+:::
+
+### Image Tag to Use
+
+:::caution Image Security
+This part of the TRG contributes to the security of your application's container image and helps to keep the basis of
+the image as up-to-date as possible.
+:::
+
+To keep application images compliant, up-to-date and secure it is mandatory to use image tags which allows to rebuild
+the applications container image without touching the `DOCKERFILE` each time to adjust the version of the base image (
+e.g. to fix vulnerabilities in the base image, see [Base Image Security](#base-image-security)).
+
+Whenever possible, do not use an image tag containing a patch version, instead, use an image tag containing either minor
+or major version:
+
+:::warning Do not use image tag with patch version
+
+```DOCKERFILE
+FROM alpine:3.18.5 as BUILDER
+
+COPY . /app/build
+WORKDIR /app/build
+RUN build
+
+FROM alpine:3.18.5
+
+COPY --from=BUILDER /app/build /app
+[...]
+```
+
+:::
+
+:::tip Use image tag with minor or major version
+
+```DOCKERFILE
+FROM alpine:3 as BUILDER
+
+COPY . /app/build
+WORKDIR /app/build
+RUN build
+
+FROM alpine:3.18
+
+COPY --from=BUILDER /app/build /app
+
+[...]
+```
+
+:::
+
+Using minor image tags instead of patch level image tags avoid to touch the applications `DOCKERFILE` to fix a
+potential problem inside the base image. With the next release of the Eclipse-Tractus-X application/component an updated
+base image will be used for the application, if available.
+
+### Base Image Security
+
+We trust in the maintenance and security mechanisms of the base images which are aligned with this TRG (see [table
+above](#base-images-to-use)). Don't try to fix vulnerabilities in base images. Instead, we focus ourselves to keep our
+code free of vulnerabilities.
+
+You shall not change the content of the base image to fix vulnerabilities of the base image using package
+manager update mechanisms inside a `DOCKERFILE`:
+
+```DOCKERFILE
+FROM alpine:3.18.2
```

Comment:
as a good example would also remove the patch version information

Reply:
Get you point, but this is an example using patchlevel image tag **and** updating/chaning the base image using package manager, so I would stay with this example.

Reply:
Got your point, my intention was ... do you really want to separate the "bad" examples, one for patchlevel "Do not use image tag with patch version" and another example for "updating/chaning the base image" **and** "example using patchlevel image"?

Maybe you can combine it in a bigger example in the `warning` section and this would also keep it simple, when decide to one TRG update instead of separating it.